### PR TITLE
Autoupdater disabled message

### PR DESCRIPTION
**Pull request by rubo77**
If the Auto Updater is disabled, this PR adds a message

>    Automatic updates are disabled. They can be enabled in Advanced Settings

### CODE REVIEW

File: package/gluon-config-mode-autoupdater/i18n/de.po [hunk 1]
```diff
@@ -13,5 +13,13 @@ msgstr ""
...
 "This node will automatically update its firmware when a new version is "
 "available."
-msgstr "Dieser Knoten aktualisiert seine Firmware automatisch, sobald "
+msgstr ""
+"Dieser Knoten aktualisiert seine Firmware automatisch, sobald "
 "eine neue Version vorliegt."
+
+msgid ""
+"Automatic updates are disabled. They can be enabled in <em>Advanced "
+"Settings</em>"
```
Comment (NeoRaider) [inline package/gluon-config-mode-autoupdater/i18n/de.po]: Please change "Settings" to "settings", as written on the link. We won't change the link text, a link/button is not a title and thus not title cased.

File: package/gluon-config-mode-autoupdater/i18n/de.po [hunk 2]
```diff
@@ -13,5 +13,13 @@ msgstr ""
...
+msgstr ""
+"Automatische Updates sind deaktiviert. Unter <em>Erweiterte "
+"Einstellungen</em> kannst du diese aktivieren."
```
Comment (NeoRaider) [inline package/gluon-config-mode-autoupdater/i18n/de.po]: Klingt seltsam. Besser: "Sie können in den *Erweiterten Einstellungen* aktiviert werden."

File: package/gluon-config-mode-autoupdater/luasrc/lib/gluon/config-mode/wizard/0050-autoupdater-info.lua [hunk 3]
```diff
@@ -9,6 +9,9 @@ function M.section(form)
   if enabled then
     local s = form:section(cbi.SimpleSection, nil,
       i18n.translate('This node will automatically update its firmware when a new version is available.'))
+  else
+    local s = form:section(cbi.SimpleSection, nil,
+      i18n.translate("Automatic updates are disabled. They can be enabled in <em>Advanced Settings</em>"))
```
Comment (NeoRaider) [inline package/gluon-config-mode-autoupdater/luasrc/lib/gluon/config-mode/wizard/0050-autoupdater-info.lua]: Have you checked that HTML markup is working as expected in a section description?

File: package/gluon-config-mode-autoupdater/luasrc/lib/gluon/config-mode/wizard/0050-autoupdater-info.lua [hunk 4]
```diff
@@ -9,6 +9,9 @@ function M.section(form)
   if enabled then
     local s = form:section(cbi.SimpleSection, nil,
       i18n.translate('This node will automatically update its firmware when a new version is available.'))
+  else
+    local s = form:section(cbi.SimpleSection, nil,
```
Comment (NeoRaider) [inline package/gluon-config-mode-autoupdater/luasrc/lib/gluon/config-mode/wizard/0050-autoupdater-info.lua]: Huh, why do we even assign this to a variable... Please remove the `local s = ` (in the *if* branch too).

File: package/gluon-config-mode-autoupdater/luasrc/lib/gluon/config-mode/wizard/0050-autoupdater-info.lua (hunk 3 above)
Comment (rubo77) [inline package/gluon-config-mode-autoupdater/luasrc/lib/gluon/config-mode/wizard/0050-autoupdater-info.lua]: I don't remember, but I guess, i tested it. 
it already works for example in https://github.com/freifunk-gluon/gluon/blob/master/package/gluon-config-mode-contact-info/i18n/gluon-config-mode-contact-info.pot#L9

**Comment by rubo77** [comment]
what is there to be done here? I don't quite understand the conflict. can you resolve this?

**Comment by NeoRaider** [comment]
You need to rebase the commit to solve the conflict.

**Comment by rubo77** [comment]
I rebased it to the 2016.2.x branch, where it was developed by then. When this was developed, the now 2016.2.x branch was still master, so this would have to be merged to the 2016.2.x branch

the adaptions to gluon-web would be a totally different file

**Comment by NeoRaider** [comment]
Ah, I didn't see that this was against v2016.2.x. Please make a new PR against the master, we never merge changes to v2016.2.x that aren't in master yet. I don't think we'll accept it in v2016.2.x even after that, as v2016.2.6 will very likely be the last v2016.2.x release.

**Comment by rubo77** [comment]
I created a local PR in my fork. maybe you can have alook, if I could create this as PR here, although I have not tested it?

https://github.com/rubo77/gluon/pull/4/files#diff-d14fa3984173ccbc7d533d63ab1472d8

**Comment by rubo77** [comment]
new PR for master: #1137

**Pull request closed without merging**